TrafficCapture/captureEndToEndDocker/build.gradle

```diff
@@ -0,0 +1,94 @@
+plugins {
+    id 'org.opensearch.migrations.java-library-conventions'
+    id "com.avast.gradle.docker-compose" version "0.16.12"
+    id "com.bmuschko.docker-java-application" version "9.3.1"
+}
+
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
+dependencies {
+    implementation project(':trafficCaptureProxyServer')
+    implementation project(':trafficReplayer')
+}
+
+
+def containerServices = [
+        "trafficCaptureProxyServer": "capture_proxy",
+        "trafficReplayer": "traffic_replayer"
+]
+containerServices.each { projectName, dockerImageName ->
+    def dockerBuildDir = "${project}/docker/${projectName}"
+    def artifactsDir = "${dockerBuildDir}/jars";
+    task("copyArtifact_${projectName}", type: Copy) {
+        dependsOn ":${projectName}:build"
+        dependsOn ":${projectName}:jar"
+        from { project(":${projectName}").configurations.findByName("runtimeClasspath").files }
+        from { project(":${projectName}").tasks.getByName('jar') }
+        into artifactsDir
+        include "*.jar"
+        duplicatesStrategy = DuplicatesStrategy.WARN
+    }
+
+    task "createDockerfile_${projectName}"(type: com.bmuschko.gradle.docker.tasks.image.Dockerfile) {
+        dependsOn "copyArtifact_${projectName}"
+        destFile = project.file("${dockerBuildDir}/Dockerfile")
+        from 'openjdk:11-jre'
```

Comment:
It's a style issue.  It may be able to be moved up to an import as long as there isn't a conflict, though that might be why it was a fully-qualified reference in the first place, as "Dockerfile" is pretty generic.

Reply:
It's a style issue.  It may be able to be moved up to an import as long as there isn't a conflict, though that might be why it was a fully-qualified reference in the first place, as "Dockerfile" is pretty generic.